### PR TITLE
[CLOUDGA-13717] Add CLI support for deleting a custom role

### PR DESCRIPTION
**Pull request by harshitha-yb**
Screenshot 2023-05-17 at 1 06 07 AM

### CODE REVIEW

File: internal/client/client.go [hunk 1]
```diff
@@ -775,6 +775,39 @@ func (a *AuthApiClient) ListTasks() ybmclient.ApiListTasksRequest {
 	return a.ApiClient.TaskApi.ListTasks(a.ctx, a.AccountID)
 }
 
+func (a *AuthApiClient) ListRbacRoles() ybmclient.ApiListRbacRolesRequest {
```
Comment (posriniv) [inline internal/client/client.go:778]: nit: this function is repeated in another PR, please take care of it while merging.

**Review by posriniv — APPROVED** [review]
Looks good to me!

File: cmd/role/role.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	ybmAuthClient "github.com/yugabyte/ybm-cli/internal/client"
+	"github.com/yugabyte/ybm-cli/internal/formatter"
+)
+
+var RoleCmd = &cobra.Command{
+	Use:   "role",
+	Short: "Manage roles",
+	Long:  "Manage roles",
+	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Help()
```
Comment (cdavid) [inline cmd/role/role.go:34]: `make doc`

**Comment by harshitha-yb** [comment]
Have created a unified PR

**Pull request closed without merging**